### PR TITLE
Docs: align docs with current repo state

### PR DESCRIPTION
**Pull request by liobrasil**
Align documentation with the current contracts/tests/transactions for audit readiness.

Changes:
- Update README + testing instructions
- Update docs (auto-borrowing, liquidation design, deposit capacity)
- Mark legacy contract file as reference-only
- Remove obsolete milestone doc

### CODE REVIEW

**Review by zhangchiqing — APPROVED** [review]

File: cadence/tests/TEST_COVERAGE.md [hunk 1]
```diff
@@ -1,16 +1,63 @@
...
 
+## Historical Narrative (May Be Outdated)
+
+The sections below were written as a narrative-style coverage analysis (original draft date: 2026-01-28).
+Treat any specific coverage percentages, repository labels, and file counts below as non-authoritative unless re-validated.
+
 ## Executive Summary
 
-This document provides a comprehensive analysis of the existing test coverage for the FlowALPv1 lending protocol, an automated lending system similar to Aave built on the Flow blockchain using the Cadence programming language. The protocol has achieved 89.7% code coverage with 31 core test files covering fundamental operations, interest mechanics, liquidations, and edge cases.
+This document provides a narrative-style analysis of the existing test coverage for the FlowALPv1 lending protocol, an automated lending system similar to Aave built on the Flow blockchain using the Cadence programming language. The protocol has an extensive test suite covering fundamental operations, interest mechanics, liquidations, and edge cases.
```
Comment (Kay-Zee) [inline cadence/tests/TEST_COVERAGE.md]: i agree with not using comprehensive, but what does it mean for it to be a narrative style analysis?

File: README.md [hunk 2]
```diff
@@ -88,54 +98,69 @@ For detailed test status and FlowVault removal summary, see [TestingCompletionSu
...
+cd FlowALP
 git submodule update --init --recursive
 ```
 
-2. Install dependencies:
+2. **Install dependencies (REQUIRED):**
+
 ```bash
 flow deps install
-cd FlowActions && flow deps install && cd ..
```
[the remark this answers is not in the document]
Comment (liobrasil) [reply README.md]: I needed to install the dependencies for the tests to run

File: cadence/tests/TEST_COVERAGE.md (hunk 1 above)
Comment (liobrasil) [reply cadence/tests/TEST_COVERAGE.md]: I've updated this to be clearer for the reader

Comment (liobrasil) [reply cadence/tests/TEST_COVERAGE.md]: I've used `walkthrough of the existing...` instead
